PR-CHECKLIST

```diff
@@ -0,0 +1,19 @@
+# Checklist for merging PRs
+
+Before merging a PR the PR should follow these rules:
+
+* The PR does not have the label "hold"
+* There is an associated GitHub Issue linked to the PR
```

Comment:
Still don't quite agree with this, we certainly have not been doing this for everything in the Admin Console. Especially for quick fixes it's just extra bookkeeping.

Reply:
Not having an issue for a fix results in it not being included in changelogs for the release, it makes it harder to backport an issue, plan, etc.

Then if an issue is not fixed by a PR, for example if the PR is closed, then you don't have an issue to capture what was broken.

I can appreciate when sending a PR for a fix creating the issue can be seen as overhead, but for everyone else not having the issue becomes cumbersome.

Reply:
Added an exception where GitHub Issue is not required. However, if a PR introduces a feature or enhancement, or is fixing an bug where previously released versions of Keycloak are affected we need the GitHub Issue.

Reply:
> Added an exception where GitHub Issue is not required. However, if a PR introduces a feature or enhancement, or is fixing an bug where previously released versions of Keycloak are affected we need the GitHub Issue.

That sounds reasonable thanks.